src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -694,6 +694,8 @@ private boolean addStaticCodeAnalysisFeedbackToResult(Result result, List<Static
             return false;
         }
         var feedbackList = feedbackService.createFeedbackFromStaticCodeAnalysisReports(reports);
+        log.debug("Created static code analysis feedback for participation " + result.getParticipation().getId()
```

Comment:
Do we need this log? it could be very spammy

Reply:
Indeed, removing this